storybook/PageOverlayPanel.qml

```diff
@@ -0,0 +1,173 @@
+import QtQuick
+import QtQuick.Layouts
+import QtQuick.Controls
+
+import StatusQ.Core.Theme
+
+Control {
+    id: root
+
+    property int style // Theme.Style
+    property int fontSizeOffset
+    property real themePadding
+
+    signal styleRequested(int style) // Theme.Style
```

Comment:
```suggestion
    signal styleRequested(int style) // ThemeUtils.Style
```

Reply:
Actually there is `Theme.Style` used indeed.